pkg/nodes/node.go

```diff
@@ -14,6 +15,11 @@ import (
 	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
```

Comment:
Please move this consts to ./pkg/nodes/nodes.go

Reply:
done